SYCL/Regression/2020-spec-constants-debug-info.cpp

```diff
@@ -0,0 +1,29 @@
+// RUN: %clangxx -fsycl -g %s -o %t.out
+// RUN: %clangxx -fsycl -g -O0 %s -o %t.out
+// RUN: %clangxx -fsycl -g -O2 %s -o %t.out
+//
+// The idea of this test is to make sure that we can compile the following
+// simple example without crashes/assertions firing at llvm-spirv step due to
+// debug info corrupted by sycl-post-link
+
+#include <CL/sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
Shouldn't it be SYCL 2020 compliant?

Reply:
Done in 2472f375a22dd0171794d28938b3182393840301